extensions/wikia/AuthModal/js/AuthModal.js

```diff
@@ -67,12 +75,15 @@ define('AuthModal', ['jquery', 'wikia.window'], function ($, window) {
 		}
 	}
 
-	function loadPage (url, callback) {
-		var authIframe = window.document.createElement('iframe');
-		authIframe.src = url + '&modal=1';
+	function loadPage (url, onPageLoadedCallback) {
```

Comment:
I don't think we need a word "callback" here. onPageLoaded is sufficient enough :)